.changeset/wicked-starfishes-shop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lottiefiles/dotlottie-svelte': major
```

Comment:
@moonlitgrace just a small thing i've noticed. 
I think this should be minor, not major since we're not introducing any breaking changes. 
Thanks again

Reply:
Oh sorry I didn't notice that step in prompt, anyways thank you @theashraf for your time.
Resolved on https://github.com/LottieFiles/dotlottie-web/pull/217/commits/cf971dfc0e3614d84e3a5a69c73e048e0ee110bd